src/aoevidencedisplay.cpp

```diff
@@ -47,7 +47,16 @@ void AOEvidenceDisplay::show_evidence(QString p_evidence_image,
   evidence_icon->resize(f_pixmap.size());
   evidence_icon->move(icon_dimensions.x, icon_dimensions.y);
 
-  evidence_movie->play(gif_name);
+  AOLayer::LayerProperties evi_props{
+      .type = AOLayer::LayerType::ui,
+      .filename = gif_name,
+      .charname = "",
+      .miscname = "",
+      .static_duration = 320,
+      .max_duration = 1000,
+  };
+  evidence_movie->set_play_once(true);
```

Comment:
What's the point of this setter when you already have LayerProperties to decide when something should play once?